### ISSUE TITLE
Multiple day spanning events

### ISSUE THREAD

**Issue by i-ed** [themouette/jquery-week-calendar#78]
Are we able to have an event which spans across multiple days?

**Comment by K-Phoen** [comment]
Yes, week-calendar handles this natively. You just have to define the start and end dates as usual and it will work.

**Comment by themouette** [comment]
Hi,

there is an issue dealing with multi day events, especially it is not possible to drag/drop multi day events.
if anyone feels like fixing this or adding a space to display it in the header, he is welcome.

and by the way, issues are not here for questions, use the mailing list instead.

Julien

**Comment by i-ed** [comment]
Tried that but dosnt work for me.

On 15/08/2011 14:45, K-Phoen wrote:

> Yes, week-calendar handles this natively. You just have to define the start and end dates as usual and it will work.

## 

Ash

**Comment by i-ed** [comment]
Well it is sort of an issue as from what ive tried i cannot get it to 
work accross multiple days...

On 15/08/2011 15:06, themouette wrote:

> Hi,
> 
> there is an issue dealing with multi day events, especially it is not possible to drag/drop multi day events.
> if anyone feels like fixing this or adding a space to display it in the header, he is welcome.
> 
> and by the way, issues are not here for questions, use the mailing list instead.
> 
> Julien

## 

Ash

**Comment by adammeghji** [comment]
Hi,

i-ed, is your event spanning multiple days spanning Sunday at Midnight?  If so, there's a bug in HEAD which causes events that span this point in the week to not appear at all.

I have submitted a pull request (https://github.com/themouette/jquery-week-calendar/pull/82) which fixes this -- hopefully that solves your problem too!

Cheers,
Adam
